publications/abstracts/reservoir-computing-abs/background.tex

```diff
@@ -0,0 +1,45 @@
+\section{Background}
+Variability has been the primary drawback for renewable energy sources like
+wind turbines, solar PV, and solar concentrators since their inception. This
+flaw has become more pronounced as renewable penetration on the electricity
+grid increased in recent years. Forecasting electricity production from
+renewable sources is therefore important for successful management of power
+systems \cite{kobylinski_high-resolution_2020}. Recent studies have applied
+\glspl{ann}, specifically multi-layer perceptrons, to the task of net load
+forecasting \cite{kobylinski_high-resolution_2020,dutta_load_2017,lee_development_2016}. These studies made short
+term forecasts of 4-6 hours. Forecasts at this time scale enable load following
+for natural gas plants, but challenges nuclear power plants. Nuclear plants
+need accurate forecasts further ahead to facilitate relaxed load following.
+This study will be the first to apply \glspl{ESN} to the task of net load
+prediction.
+
+The University of Illinois at Urbana-Champaign is an ideal model system for
+this work because of its diverse energy mix. Previous work has been done to
```

Comment:
I think saying a diverse mix is a bit redundant. Maybe say diverse energy sources?